content/en/database_monitoring/guide/sql_deadlock.md

```diff
@@ -49,7 +50,7 @@ Supported Agent versions
   GO
 ```
 
-2. In the Datadog Agent, enable deadlocks in `sqlserver.d/conf.yaml`.
+2. In addition to the [DBM for SQL Server setup][1]. In the Datadog Agent, enable deadlocks in `sqlserver.d/conf.yaml`.
```

Comment:
I would make this clarification in a different place. It belongs in a prerequisites section, not inline in the procedure.

I suggest:

* Get rid of this line change
* Get rid of line 18
* Under line 20 (`## Before you begin`), add a line that says something like `You must configure Database Monitoring for your [SQL Server][1] before following the steps in this guide.`

I've added these as suggested edits.